npm/config/config.go

```diff
@@ -3,12 +3,13 @@ package npmconfig
 import "github.com/Azure/azure-container-networking/npm/util"
 
 const (
-	defaultResyncPeriod    = 15
-	defaultApplyMaxBatches = 100
-	defaultApplyInterval   = 500
-	defaultListeningPort   = 10091
-	defaultGrpcPort        = 10092
-	defaultGrpcServicePort = 9002
+	defaultResyncPeriod         = 15
+	defaultApplyMaxBatches      = 100
+	defaultApplyInterval        = 500
+	defaultMaxBatchedACLsPerPod = 100
```

Comment:
Lets make this smaller, if there is an issue with applying ACLs due to some weird limit of data type they are using underneath, we can end up in a limbo of not being able to apply a bunch of network policies at once. 

Smaller means more network policies are guaranteed to get applied. Wdyt ?

Reply:
discussed offline to make this 30